01:00:24 PM:DEBUG [logger_base.py:15] Mensaje a nivel debug
01:00:24 PM:INFO [logger_base.py:16] Mensaje a nivel info
01:00:24 PM:WARNING [logger_base.py:17] Mensaje a nivel warning
01:00:24 PM:ERROR [logger_base.py:18] Mensaje a nivel error
01:00:24 PM:CRITICAL [logger_base.py:19] Mensaje a nivel critical
12:44:22 PM:DEBUG [Persona.py:53] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
12:44:40 PM:DEBUG [Persona.py:53] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
        
12:49:24 PM:DEBUG [Persona.py:53] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
        
12:55:39 PM:DEBUG [Persona.py:53] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
        
12:55:39 PM:DEBUG [Persona.py:55] 
            Id Persona: None, 
            Nombre: Jose, 
            Apellido: Lopez, 
            Email: ljose@mail.com
        
12:55:39 PM:DEBUG [Persona.py:57] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
        
12:56:05 PM:DEBUG [Persona.py:53] 
            Id Persona: 1, 
            Nombre: Juan, 
            Apellido: Perez, 
            Email: jperez@mail.com
        
12:56:05 PM:DEBUG [Persona.py:55] 
            Id Persona: None, 
            Nombre: Jose, 
            Apellido: Lopez, 
            Email: ljose@mail.com
        
12:56:05 PM:DEBUG [Persona.py:57] 
            Id Persona: 1, 
            Nombre: None, 
            Apellido: None, 
            Email: None
        
12:43:42 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000002092C8C3120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:47:07 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:47:47 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001FF8F753120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:47:55 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:49:04 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:49:24 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:53:34 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000028456D23120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:53:55 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:54:19 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:55:53 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001CAD9113120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:55:53 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001CAD9713840; closed: 0>
12:57:16 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:57:29 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:57:48 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:57:48 PM:ERROR [Conexion.py:27] Ocurrio un error: 
12:57:48 PM:ERROR [Conexion.py:40] Ocurrio un error: 'NoneType' object has no attribute 'cursor'
12:58:57 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000002228D6D3120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:58:57 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002228E063840; closed: 0>
01:00:15 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:00:15 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:00:15 PM:ERROR [Conexion.py:40] Ocurrio un error: 'NoneType' object has no attribute 'cursor'
01:01:15 PM:ERROR [Conexion.py:27] Ocurri un error: 
01:01:40 PM:ERROR [Conexion.py:27] Ocurri un error: 
01:02:05 PM:ERROR [Conexion.py:27] 
01:02:32 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:02:37 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:04:20 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000024DD3E23120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:04:20 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000024DD4773840; closed: 0>
01:04:33 PM:ERROR [Conexion.py:27] Ocurrio un error: 
Traceback (most recent call last):
  File "C:\Users\gusta\TecnicaturaGit\python3\Leccion7\capa_datos_persona\Conexion.py", line 19, in obtenerConexion
    cls._conexion = bd.connect(host=cls._HOST,
  File "C:\Users\gusta\TecnicaturaGit\python3\Leccion7\venv\lib\site-packages\psycopg2\__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError
01:07:46 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:09:30 PM:ERROR [Conexion.py:27] Ocurrio un error: 
01:10:02 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000227498C3120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:10:02 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000022749EC3840; closed: 0>
10:13:32 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000227FD573120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:13:32 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x00000227FDB73920; closed: 0>
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 5, 
            Nombre: Juan Carlos, 
            Apellido: Roldan, 
            Email: jcroldan@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 6, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 8, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 9, 
            Nombre: Marcos, 
            Apellido: Canto, 
            Email: mcanto@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 10, 
            Nombre: Marcelo, 
            Apellido: Cuenca, 
            Email: cuencam@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 11, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 12, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 13, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 14, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
10:13:32 PM:DEBUG [Persona_dao.py:31] 
            Id Persona: 15, 
            Nombre: Alex, 
            Apellido: Rojas, 
            Email: arojas@mail.com
        
10:39:06 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001B5022F3120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:39:06 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001B5028F3920; closed: 0>
10:39:06 PM:DEBUG [Persona_dao.py:37] Persona insertada: 
            Id Persona: None, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
10:39:06 PM:DEBUG [Persona_dao.py:44] Personas insertadas: 1
10:41:02 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001FF8CD63120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:41:02 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001FF8D363920; closed: 0>
10:41:02 PM:DEBUG [Persona_dao.py:37] Persona insertada: 
            Id Persona: None, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
10:41:02 PM:DEBUG [Persona_dao.py:44] Personas insertadas: 1
10:42:17 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000002E1882B3120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:42:17 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002E1888B3920; closed: 0>
10:42:17 PM:DEBUG [Persona_dao.py:37] Persona insertada: 
            Id Persona: None, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
10:42:17 PM:DEBUG [Persona_dao.py:44] Personas insertadas: 1
10:44:25 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000029C27703120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:44:25 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000029C27D03920; closed: 0>
10:44:25 PM:DEBUG [Persona_dao.py:37] Persona insertada: 
            Id Persona: None, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
10:44:25 PM:DEBUG [Persona_dao.py:44] Personas insertadas: 1
10:49:53 PM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000026999A23120; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
10:49:53 PM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002699A023920; closed: 0>
10:49:53 PM:DEBUG [Persona_dao.py:37] Persona insertada: 
            Id Persona: None, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
10:49:53 PM:DEBUG [Persona_dao.py:44] Personas insertadas: 1
12:42:54 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000013DF3792570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:42:54 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000013DF58A9540; closed: 0>
12:44:43 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000208467D2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:44:43 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000020848C99540; closed: 0>
12:46:00 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000019179452570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:46:00 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001917B569540; closed: 0>
12:47:26 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001CF737C2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:47:26 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001CF75C69540; closed: 0>
12:50:17 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000024C091B2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:50:17 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000024C0B2C9540; closed: 0>
12:52:05 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000016CEEF62570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:52:05 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000016CF1079540; closed: 0>
12:52:05 AM:DEBUG [Persona_dao.py:49] Persona actualizada: 
            Id Persona: 5, 
            Nombre: Juan Jose, 
            Apellido: Pena, 
            Email: jjpena@mail.com
        
12:52:05 AM:DEBUG [Persona_dao.py:56] Personas actualizadas: 1
01:07:29 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001920B792570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:07:29 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001920D8A9540; closed: 0>
01:07:29 AM:DEBUG [Persona_dao.py:59] Persona eliminada: 
            Id Persona: 5, 
            Nombre: None, 
            Apellido: None, 
            Email: None
        
01:07:29 AM:DEBUG [Persona_dao.py:66] Personas eliminadas: 1
01:14:24 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001D436A32570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:14:24 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001D438B79540; closed: 0>
01:41:53 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000189FCB32570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:41:53 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x00000189FF009540; closed: 0>
01:44:45 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000026259AA2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:44:45 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002625BF69540; closed: 0>
01:44:45 AM:DEBUG [Persona_dao.py:59] Persona eliminada: 
            Id Persona: 10, 
            Nombre: None, 
            Apellido: None, 
            Email: None
        
01:44:45 AM:DEBUG [Persona_dao.py:66] Personas eliminadas: 1
01:54:32 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000178AA342570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:54:32 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x00000178AC459540; closed: 0>
01:56:49 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000012834DD2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:56:49 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000012836F59540; closed: 0>
01:56:49 AM:DEBUG [Persona_dao.py:59] Persona eliminada: 
            Id Persona: 11, 
            Nombre: None, 
            Apellido: None, 
            Email: None
        
01:56:49 AM:DEBUG [Persona_dao.py:67] Personas eliminadas: 1
01:58:12 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000216E3F12570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:58:12 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x00000216E6029540; closed: 0>
01:58:34 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000002BF2A592570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
01:58:34 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002BF2C6A9540; closed: 0>
02:05:18 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x00000180B0512570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:05:18 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x00000180B2629540; closed: 0>
02:05:42 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000027126432570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:05:42 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000027128919540; closed: 0>
02:07:26 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001D3F9B42570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:07:26 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001D3FBFF9540; closed: 0>
02:07:26 AM:DEBUG [Persona_dao.py:61] Persona eliminada: (12, 'Maria', 'Esparza', 'mesparza@mail.com')
02:07:26 AM:DEBUG [Persona_dao.py:69] Personas eliminadas: 1
02:07:55 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000001F2F7FC2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:07:55 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000001F2FA119540; closed: 0>
02:07:55 AM:DEBUG [Persona_dao.py:61] Persona eliminada: None
02:07:55 AM:DEBUG [Persona_dao.py:69] Personas eliminadas: 0
02:08:22 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x000002E012852570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:08:22 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x000002E014D29540; closed: 0>
02:08:22 AM:DEBUG [Persona_dao.py:61] Persona eliminada: (15, 'Alex', 'Rojas', 'arojas@mail.com')
02:08:22 AM:DEBUG [Persona_dao.py:69] Personas eliminadas: 1
02:14:49 AM:DEBUG [Conexion.py:24] Conexion Exitosa: <connection object at 0x0000015025BD2570; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:14:49 AM:DEBUG [Conexion.py:37] Se abrio correctamente el cursor: <cursor object at 0x0000015027CF9540; closed: 0>
02:14:49 AM:DEBUG [Persona_dao.py:61] Persona eliminada: (20, 'Pedro', 'Romero', 'promero@mail.com')
02:14:49 AM:DEBUG [Persona_dao.py:70] Personas eliminadas: 1
11:38:46 AM:DEBUG [Conexion.py:67] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x000001D9CB0A2770>
11:39:35 AM:DEBUG [Conexion.py:67] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000017D9B642800>
11:39:35 AM:DEBUG [Conexion.py:22] Conexion obtenida del pool: <connection object at 0x0000017D993A3230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
11:39:35 AM:DEBUG [Conexion.py:22] Conexion obtenida del pool: <connection object at 0x0000017D993A3010; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
11:41:05 AM:DEBUG [Conexion.py:67] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000024CDDC32890>
11:41:05 AM:DEBUG [Conexion.py:22] Conexion obtenida del pool: <connection object at 0x0000024CDBD23230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
11:41:05 AM:DEBUG [Conexion.py:22] Conexion obtenida del pool: <connection object at 0x0000024CDBD23010; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
11:41:05 AM:DEBUG [Conexion.py:22] Conexion obtenida del pool: <connection object at 0x0000024CDBD23EF0; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:42:00 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000011A6FFEBC70>
02:42:00 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x0000011A6DE53230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:42:00 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x0000011A6DE53230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:45:58 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x00000131EB0871C0>
02:45:58 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x00000131E9163230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
02:45:58 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x00000131E9163230; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:22:58 PM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
08:22:58 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000016AA3777EE0>
08:22:58 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x0000016AA5354040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:22:58 PM:DEBUG [cursor_del_pool.py:28] Dentro del bloque with
08:22:58 PM:DEBUG [cursor_del_pool.py:30] [(6, 'Carlos', 'Lara', 'clara@mail.com'), (8, 'Carlos', 'Lara', 'clara@mail.com'), (9, 'Marcos', 'Canto', 'mcanto@mail.com'), (13, 'Maria', 'Esparza', 'mesparza@mail.com'), (14, 'Carlos', 'Lara', 'clara@mail.com'), (16, 'Pedro', 'Romero', 'promero@mail.com'), (17, 'Pedro', 'Romero', 'promero@mail.com'), (18, 'Pedro', 'Romero', 'promero@mail.com'), (19, 'Pedro', 'Romero', 'promero@mail.com')]
08:22:58 PM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
08:22:58 PM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
08:22:58 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x0000016AA5354040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:34:03 PM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
08:34:03 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x000001FEFD127EE0>
08:34:03 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x000001FEFECB0040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:34:03 PM:DEBUG [cursor_del_pool.py:28] Dentro del bloque with
08:34:03 PM:DEBUG [cursor_del_pool.py:30] [(6, 'Carlos', 'Lara', 'clara@mail.com'), (8, 'Carlos', 'Lara', 'clara@mail.com'), (9, 'Marcos', 'Canto', 'mcanto@mail.com'), (13, 'Maria', 'Esparza', 'mesparza@mail.com'), (14, 'Carlos', 'Lara', 'clara@mail.com'), (16, 'Pedro', 'Romero', 'promero@mail.com'), (17, 'Pedro', 'Romero', 'promero@mail.com'), (18, 'Pedro', 'Romero', 'promero@mail.com'), (19, 'Pedro', 'Romero', 'promero@mail.com')]
08:34:03 PM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
08:34:03 PM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
08:34:03 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x000001FEFECB0040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:50:15 PM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
08:50:15 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000017E3A5A8FA0>
08:50:15 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x0000017E3A564040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:50:15 PM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
08:50:15 PM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
08:50:15 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x0000017E3A564040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 6, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 8, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 9, 
            Nombre: Marcos, 
            Apellido: Canto, 
            Email: mcanto@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 13, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 14, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 16, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 17, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 18, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:50:15 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 19, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:56:57 PM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
08:56:57 PM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x0000021BC77569E0>
08:56:57 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x0000021BC7744040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:56:57 PM:DEBUG [Persona_dao.py:38] Persona insertada: 
            Id Persona: None, 
            Nombre: Marcos, 
            Apellido: Tejada, 
            Email: mtejada@mail.com
        
08:56:57 PM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
08:56:57 PM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
08:56:57 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x0000021BC7744040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:56:57 PM:DEBUG [Persona_dao.py:78] Personas insertadas: 1
08:56:57 PM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
08:56:57 PM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x0000021BC7744040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:56:57 PM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
08:56:57 PM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
08:56:57 PM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x0000021BC7744040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 6, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 8, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 9, 
            Nombre: Marcos, 
            Apellido: Canto, 
            Email: mcanto@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 13, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 14, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 16, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 17, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 18, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 19, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
08:56:57 PM:DEBUG [Persona_dao.py:83] 
            Id Persona: 21, 
            Nombre: Marcos, 
            Apellido: Tejada, 
            Email: mtejada@mail.com
        
12:16:21 AM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
12:16:21 AM:DEBUG [Conexion.py:75] Creacion de pool de conexiones exitosa <psycopg2.pool.SimpleConnectionPool object at 0x00000208E0922E60>
12:16:21 AM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:58] Persona eliminada: (21, 'Marcos', 'Tejada', 'mtejada@mail.com')
12:16:21 AM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
12:16:21 AM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
12:16:21 AM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:67] Personas eliminadas: 1
12:16:21 AM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
12:16:21 AM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:47] Persona actualizada: 
            Id Persona: 19, 
            Nombre: Arduizur, 
            Apellido: Glickman, 
            Email: aglickman@mail.com
        
12:16:21 AM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
12:16:21 AM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
12:16:21 AM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:72] Personas actualizadas: 1
12:16:21 AM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
12:16:21 AM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:38] Persona insertada: 
            Id Persona: None, 
            Nombre: Roman, 
            Apellido: Riquelme, 
            Email: rriquelme@mail.com
        
12:16:21 AM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
12:16:21 AM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
12:16:21 AM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:78] Personas insertadas: 1
12:16:21 AM:DEBUG [cursor_del_pool.py:10] Inicio del metodo with y __enter__
12:16:21 AM:DEBUG [Conexion.py:25] Conexion obtenida del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [cursor_del_pool.py:16] Se ejecuta el metodo __exit__
12:16:21 AM:DEBUG [cursor_del_pool.py:22] Commit de la transaccion
12:16:21 AM:DEBUG [Conexion.py:88] Regresamos la conexion del pool: <connection object at 0x00000208E0910040; dsn: 'user=postgres password=xxx dbname=test_bd host=localhost port=5432', closed: 0>
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 6, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 8, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 9, 
            Nombre: Marcos, 
            Apellido: Canto, 
            Email: mcanto@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 13, 
            Nombre: Maria, 
            Apellido: Esparza, 
            Email: mesparza@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 14, 
            Nombre: Carlos, 
            Apellido: Lara, 
            Email: clara@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 16, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 17, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 18, 
            Nombre: Pedro, 
            Apellido: Romero, 
            Email: promero@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 19, 
            Nombre: Arduizur, 
            Apellido: Glickman, 
            Email: aglickman@mail.com
        
12:16:21 AM:DEBUG [Persona_dao.py:83] 
            Id Persona: 22, 
            Nombre: Roman, 
            Apellido: Riquelme, 
            Email: rriquelme@mail.com
        
